yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnSparkHadoopUtil.scala

```diff
@@ -314,5 +314,28 @@ object YarnSparkHadoopUtil {
   def getClassPathSeparator(): String = {
     classPathSeparatorField.get(null).asInstanceOf[String]
   }
+
+  /** Get the initial target number of executors depends on dynamic allocation is enabled or not */
```

Comment:
nit: "Getting the initial target number of executors depends on whether dynamic allocation is enabled."